src/main/java/fiuba/tpp/reactorapp/controller/ReactorController.java

```diff
@@ -74,6 +75,16 @@ public void deleteReactor(@PathVariable Long id){
         return reactores;
     }
 
+    @GetMapping(value = "/search")
+    public List<ReactorResponse> searchReactores(@RequestParam(required = false) Long idAdsorbato, @RequestParam(required = false) Long idAdsorbente){
+        List<ReactorResponse> reactores = new ArrayList<>();
+        ReactorFilter filter = new ReactorFilter(idAdsorbato,idAdsorbente);
+        for (Reactor reactor: reactorService.search(filter)) {
+            reactores.add(new ReactorResponse(reactor));
```

Comment:
Este for es re tonto, no se puede castear sl .search a un ArrayList más facilmente o algo así? O devuelve algo más raro el .search()?

Reply:
El For es para convertir al objeto response, se puede cambiar por alguna expresion pero la iteracion pasaria igual